### PR TITLE
Fix operator build

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

**Review by nklaassen — APPROVED** [review]

**Comment by r0mant** [comment]
Running a dev release off of branch/v12 with the same fix it looks like it does fix it: https://drone.platform.teleport.sh/gravitational/teleport/20122/27/39.

**Review by russjones — APPROVED** [review]

File: .github/workflows/unit-tests-operator.yaml [hunk 1]
```diff
@@ -20,6 +21,7 @@ on:
       - '/go.sum'
       - 'operator/**'
       - 'api/types/**'
+      - 'gen/**'
       - 'lib/tbot/**'
```
Comment (hugoShaka) [inline .github/workflows/unit-tests-operator.yaml:25]: Sadly this should be `lib/**` I think as the `gen/` dependency was added by a `lib/something` edition, which was likely a dependency of `lib/tbot`.

Comment (r0mant) [reply .github/workflows/unit-tests-operator.yaml:25]: Yeah, I was also thinking we should just run operator tests on any .go source code change to be safe. It runs fast anyway. Will fix later (after release).

**Comment by ravicious** [comment]
I'm going to include this fix in my backports of #20810 to v10 and v11. Though I think v10 might not need it as `lib/services/usagereporter.go` doesn't exist on v10 but I suppose it won't hurt to backport it anyway.

**Comment by ravicious** [comment]
I'm not backporting the fix to v10 in the end because [teleport-operator builds fine on v10 without it](https://drone.platform.teleport.sh/gravitational/teleport/20199/27/39) and I don't want my PR to await a review from the admin team.

**Pull request merged**